blocks/subscriptions-block/components/useSales.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import Identity from "@arc-publishing/sdk-identity";
```

Comment:
I just noticed, `@arc-publishing/sdk-identity` isn't a dependency in the `package.json` and should be listed there.